feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -157,20 +161,49 @@ export class AddToCartComponent implements OnInit, OnDestroy {
     }
   }
 
+  refreshLiveStockData(): void {
```

Comment:
Can we change this method name to `getRealTimeStock` -> lets uniformly use `live / real-time` & `stock / stockData` - across all method /property naming

Reply:
done